src/include/homeobject/shard_manager.hpp

```diff
@@ -22,6 +22,7 @@ struct ShardInfo {
     shard_id_t id;
     pg_id_t placement_group;
     State state;
+    uint64_t lsn;
```

Comment:
Does this need to be exposed to SM or other? If not, can we hide it somewhere that's not part of the public API?

Reply:
Yeah we can make it private.

Reply:
it's okay to leave public; i was trying to not include it in the public API as moving/changing it will impact the layout/size of this structure for code that depends on homeobject (SM), requiring a re-compile rather than a simple re-link.

Reply:
Discussed with Brian. ShardInfo used in public api also used everywhere in hs_shard_manager.cpp. May be cleanup in future.